src/pages/patterns/notification-pattern/index.mdx

```diff
@@ -270,14 +254,49 @@ modal.
 </Column>
 </Row>
 
+### Actionable
+
+[Actionable](https://www.carbondesignsystem.com/components/notification/usage/)
+notifications allow for interactive elements within a notification styled like
+an inline or toast notification. Actionable notifications, since they require
+user interaction, take focus when triggered and can be highly disruptive to
+screen readers and keyboard users.
+
+Actionable notifications styled as inline notifications contain only one ghost
+button. A small “x” in the top right corner is used to dismiss inline
+notifications. Including the close button is optional and should not be included
+if it is critical for a user to read or interact with the notification.
+
+Actionable notifications styled as inline notifications can only one action
```

Comment:
> "Actionable notifications styled as inline notifications can only one action..." 

- Did you mean to say `toast` notifications here instead of inline?
- Need to add the word `have` in this sentence after "only".